scripts/dev-setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Nit: 

Putting this in the `scripts` folder makes it accessible using make, which runs everything through dapper. Since this can't be used through dapper (at least AFAIK) and needs to be called directly, it might be good to put it into another folder which isn't usable from make. Another PR had a similar issue and they created a `dev-scripts` folder (hasn't merged yet: https://github.com/rancher/rancher/pull/41216), which might serve your purposes.

Reply:
Good find, will do that, thanks